README.md

```diff
@@ -513,17 +513,29 @@ On most Linux-based systems, the HTTP API user mapping plugins will be installed
 This plugin looks for a claim that provides a direct mapping of an authenticated OpenID user to an iRODS user.
 This requires that you have the ability to add a claim within the OpenID Provider.
 
+Additionally, the plugin allows you to modify the claim found by matching the claim using a regex and replacing the claim with your
```

Comment:
Use "regular expression" instead of "regex" so there's no confusion.

Reply:
```suggestion
Additionally, the plugin allows you to modify the claim found by matching the claim using a regular expression (regex) and replacing the claim with your
```